thrift/src/main/thrift/atoms/storyquestions.thrift

```diff
@@ -32,4 +32,5 @@ struct StoryQuestionsAtom {
   4: optional list<QuestionSet> editorialQuestions
   5: optional list<QuestionSet> userQuestions
   6: optional shared.NotificationProviders notifications
+  7: optional bool closed = false
```

Comment:
This feels like something that could be offered for all atoms and therefore at the top level. I can imagine cases where this will crop up again, for example we had a case in Commercial where if an atom is pointing to some content that has expired, you would no longer want that atom to show.